controller/loginController_test.go

```
@@ -99,8 +99,8 @@ func (suite *LoginControllerTestSuite) SetupSuite() {
 
 // TearDownSuite runs once after all tests in the suite
 func (suite *LoginControllerTestSuite) TearDownSuite() {
-	if suite.sugar != nil {
-		suite.sugar.Sync()
+	if suite.logger != nil {
+		suite.logger.Sync()
```

golangci-lint (1.64.8)

Error return value of `suite.logger.Sync` is not checked

(errcheck)